liferay-frontend/liferay-portal
# LPS-120676 [Content Dashboard] Fix wrong copy message when a bar is selected

**Description**

We've confirmed that a way to zoom into one category's assets in the graph is needed in the Content Dashboard. By clicking on the bars in the chart, users will be able to filter the content's list below. The filter will consist in applying an "AND" operator with the selected category over the already existing filters.

**History**

* https://github.com/brianchandotcom/liferay-portal/commit/bdd1d74c78be557d1d358550319d3eae4c9c975a -> Compiling was broken
* https://github.com/liferay-tango/liferay-portal/commit/64ccaf095d72dc2a56133197f5abe26ebc620e2d -> This change fix the compilation problem and I tried with that key and apparently it was working fine, but false alarm, the portal seems not getting the language key 😢 

**Solution**

Language key:

```
press-escape-to-remove-the-bar's-category-filters=Press escape to remove the bar's category filters.
```

And in the JS:

```js
Liferay.Language.get("press-escape-to-remove-the-bar's-category-filters")
```

Also, I fix the typo in the language message.

**How to test it**

* Create at least one category for one vocabulary (for example: Fantasy inside Topic vocabulary)
* Create a Web Content and categorize it with Fantasy category
* Go to Content Dashboard and configure the graph with Topic vocabulary
* Click in one of the bars of the graph to see the alert info message

**Screenshot**

<img width="1239" alt="Screenshot 2021-02-22 at 16 45 17" src="https://user-images.githubusercontent.com/15845466/108731930-5c73c080-752d-11eb-92d1-5cc83b5e428f.png">
